designer/server/src/routes/forms/library.js

```diff
@@ -34,10 +37,11 @@ export default [
     path: '/library/{slug}',
     options: {
       async handler(request, h) {
-        const { params } = request
+        const { auth, params } = request
+        const token = auth.credentials.idToken
```

Comment:
`.token` not `.idToken`

Reply:
Well spotted. Due to a late refactor. I panicked for a second, wondering how is it actually working with the `idToken`, but of course that endpoint is unauthenticated.

Begs the question, should we remove the token altogether from `forms.get` here?